docs/resources/alert.md

```diff
@@ -135,8 +136,8 @@ Required:
 Optional:
 
 - `display` (String)
-- `display_type_options` (Block Set, Max: 1) Applicable options vary depending on the display type. Please see the Lightstep documentation for a full description. (see [below for nested schema](#nestedblock--composite_alert--alert--query--display_type_options))
-- `hidden_queries` (Map of String) An optional map of sub-query names in the query_string to a boolean string to hide/show that query. If specified, the map must have an entry for all named sub-queries in the query_string. A value of "true" indicates the query should be hidden. Example: `hidden_queries = {  "a" = "true",  "b" = "false" }`.
+- `display_type_options` (Block Set, Max: 1) Applicable options vary depending on the display type. Please see the Lightstep documentation for a full description. (See [below for nested schema](#nestedblock--composite_alert--alert--query--display_type_options).)
+- `hidden_queries` (Map of String) An optional map of sub alert names in the query_string to a boolean string to hide/show that query. If specified, the map must have an entry for all named sub alerts in the query_string. A value of "true" indicates the query should be hidden. Example: `hidden_queries = {  "a" = "true",  "b" = "false" }`.
```

Comment:
Three things:

- In the main docs, we decided to use sub alert (not sub-alert). I went ahead and changed things here for consistency with the main docs. I'm happy to revisit that hyphen decision whenever though.
- For consistency with the identifier for sub alerts mentioned elsewhere, I changed this from sub query to sub alert. Is that accurate? I will change it back if not. 
- Should the example here use upper case letters?

Reply:
(sub-queries are a different thing so I left those as-is)